blackjax/mcmc/metrics.py

```diff
@@ -30,21 +30,22 @@
 """
 from typing import Callable, NamedTuple, Optional, Protocol, Union
 
+import jax
 import jax.numpy as jnp
 import jax.scipy as jscipy
+from chex import Numeric
```

Comment:
Let's add the type into https://github.com/blackjax-devs/blackjax/blob/main/blackjax/types.py instead, copying the 2 lines from chex

https://github.com/google-deepmind/chex/blob/343d03af64d0ead945fdda194b9d5c5111eb31d7/chex/_src/pytypes.py#L53-L54

reason: chex usage should be test only.

Reply:
Ok